libmozevent/utils.py

```diff
@@ -139,10 +140,14 @@ class AsyncRedis(object):
     Async context manager to create a redis connection
     """
 
+    redis = contextvars.ContextVar("redis-server")  # type: aioredis.Redis
```

Comment:
Could you use the new style for type annotations?
`redis: aioredis.Redis = contextvars.ContextVar("redis-server")`